src/PowerShellEditorServices/Index.cs

```diff
@@ -0,0 +1,154 @@
+// <auto-generated>
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
I don't 100% know what to do with this and `Index`. I may have to rip these out and stop using the range syntax for arbitrary legal reasons but I'd like to check out what other MS projects are doing to polyfill.